c7n/output.py

```diff
@@ -348,7 +375,13 @@ def __exit__(self, exc_type=None, exc_value=None, exc_traceback=None):
     def join_log(self):
         self.handler = self.get_handler()
         self.handler.setLevel(logging.DEBUG)
-        self.handler.setFormatter(logging.Formatter(self.log_format))
+        try:
+            if self.ctx.options.get("log_type") == "json":
```

Comment:
why the try/except here? extending options should go to config.py 

one alternative to the log type cli parameter would be to use the existing log parameter which accepts parameterization (its a url so query `?format=json`)